app/v2/inbound_sms/inbound_sms_schemas.py

```diff
@@ -0,0 +1,77 @@
+from app.schema_validation.definitions import uuid
+
+
+get_inbound_sms_request = {
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "description": "schema for query parameters allowed when getting list of received text messages",
+    "type": "object",
+    "properties": {
+        "older_than": uuid,
+    },
+    "additionalProperties": False,
+}
+
+
+get_inbound_sms_single_response = {
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "description": "GET inbound sms schema response",
+    "type": "object",
+    "title": "GET response v2/inbound_sms",
+    "properties": {
+        "provider_date": {
+            "format": "date-time",
+            "type": "string",
+            "description": "Date+time sent by provider"
+        },
+        "provider_reference": {"type": ["string", "null"]},
```

Comment:
I doubt the provider reference is important to the service

Reply:
ok